src/components/Form.js

```diff
@@ -178,7 +178,13 @@ const Form = () => {
 
     let submission;
     try {
-      submission = await createSubmission(config.baseUrl, form, config.clientBaseUrl);
+      submission = await createSubmission(
```

Comment:
@sergei-maertens I don't think this line was covered before either, and I'm not sure where to start with writing a test for this `onFormStart` :sweat_smile: